server/controllers/notificationSettingsController.test.ts

```diff
@@ -7,17 +8,14 @@ describe('notification settings middleware', () => {
   const renderMock = jest.fn()
   const nextMock = jest.fn()
   const token = 'aubergine'
-  const csrfToken = 'courgette'
```

Comment:
fields already in `res.locals` so don't need to be mapped across in the controllers